index.html

```diff
@@ -14,8 +14,16 @@
         </header>
         <main>
             <section class="main-content">
-                <div class="worker-container"></div>
-                <div class="client-container"></div>
+                <div class="worker-container">
+                    <div class="main-logo">
+                        <img src="./img/logo-img.png" class="logo-img">
+                        <img src="./img/logo-img2.png" class="logo-img2">
+                    </div>
+                    <img src="./img/picture1.png" class="phone-img">
+                </div>
+                <div class="client-container">
+                    <img src="./img/picture2.png" class="pc-img">
+                </div>
```

Comment:
IMO
参考サイトの構造を模写するのだとすると、
main-logoは
- このページのタイトルに値するので`<h1>`
- worker-containerとclient-containerのどちらにも属していない
と思いました。

MUST
- `<img>`のalt属性は付けるべきだと思います。`<h1>`はページタイトルのロゴが画像になっていると思うので、本来はロゴの内容をそのまま入れるのですが、ダミー画像なので”サイトタイトル”だけでも入れておくと良いかと思いました。or <title>と同じ”LP制作”など。
- それ以外の装飾の画像は、クライアントから指定がない場合は `alt=""` 空のままでよいと思います。

```suggestion
                <h1 class="main-logo">
                    <img src="./img/logo-img.png" class="logo-img" alt="サイトタイトル">
                    <img src="./img/logo-img2.png" class="logo-img2" alt="サイトタイトル">
                </h1>
                <div class="worker-container">
                    <img src="./img/picture1.png" class="phone-img" alt="">
                </div>
                <div class="client-container">
                    <img src="./img/picture2.png" class="pc-img" alt="">
                </div>
```

Reply:
調べました。自分はあまり文章のないpタグの中にimgタグを入れるのは少し意味的に不自然に思えます
https://www.sitepoint.com/community/t/images-in-p-tags/5943/2
pタグの中に文章がない場合はdivにしましょうか

Reply:
承知しました。
修正しました。

062341c